docs/internal-documentation/maintenance/ReleaseProcess-test-planning.md

```diff
@@ -0,0 +1,18 @@
+Release process - Test Planning
+===============================
+
+The purpose of test planning is to raise the quality by making the release
+testing more structured, getting better coverage and finding more bugs.
```

Comment:
It does not have to be bugs (software errors), it could be spelling errors or other things. And there might not be any bugs or issues to find. I think "any issues" is better than "more bugs".

Reply:
I see your point. But I really like to have the word "more" in there to reflect that we would find some problems with unstructured testing, but we find more of them with structured testing. How do you feel about "more problems" instead of "more bugs"?

Reply:
Keep more.